std_msgs/Header header

int32 id
builtin_interfaces/Time src_time
string src_frame
uint32 max_markers
uint32 max_links
float64 confidence

Box bounding_box

Marker[] markers
Link[] links
